apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -344,10 +341,10 @@ const BookingPage = ({
       );
 
     let web3Details: Record<"userWallet" | "userSignature", string> | undefined;
-    if (eventTypeDetail.metadata.smartContractAddress) {
+    if (eventType.metadata.smartContractAddress) {
       web3Details = {
         userWallet: window.web3.currentProvider.selectedAddress,
-        userSignature: contracts[(eventTypeDetail.metadata.smartContractAddress || null) as number],
+        userSignature: contracts[(eventType.metadata.smartContractAddress || null) as number],
```

Comment:
do we keep this or accident?

Reply:
Accident, removed some more stuff too! Thanks @PeerRich